torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -463,7 +463,7 @@ Module freeze_module(const Module& module) {
         "attempted to freeze a module that return itself");
   }
 
-  auto moduleClone = module.clone();
+  auto moduleClone = module.clone(true);
```

Comment:
@bzinodev do we need to deep copy the Tensor here? I feel shallow copy is enough actually